index.md

```diff
@@ -1,51 +1,68 @@
+
+## IBM Cloud dependencies
+
+The Health Record Ingestion service was developed on the IBM Cloud and currently does not support running on other public or private clouds. However, as part of Project Alvearie, the Health Record Ingestion service is being transitioned into an open source project, with a goal of supporting public and private cloud deployments. The latest version has been pushed to GitHub. The team continues to work on moving all development activities into the open, and making the Health Record Ingestion service support other clouds. For more information about future plans, see the [Roadmap](roadmap.md).   
+
+## Core architecture
+
+<br>
+**Figure: Components of the Health Record Ingestion service**
+<br><br>
 ![core-architecture](assets/img/architecture-core.png)
+<br><br>
 
 ### Topics
-Health data, which may include [PHI](glossary.md#phi), is written to and read from the Kafka (IBM Event Streams) topics. There **_must be_** _separate topics for each tenant and Data Integrator_ in order to meet data separability requirements. 
 
-Additional topics may be created as desired. But, in general, Kafka performs better with a small number of large topics. For each (upstream) input topic, there is an associated (downstream) notification topic for [batch status notifications](https://github.com/Alvearie/hri-api-spec/tree/master/notifications/batchNotification.json). 
+Kafka (IBM Event Streams) stores streams of records in [topics](glossary.md#topic). Health data can include [Protected Health Information (PHI)](glossary.md#protected-health-information-phi). To meet data separability requirements, **there must be separate topics for each tenant and Data Integrator**. 
+
+As needed, create additional topics. Generally, Kafka performs better with a small number of large topics. For each (upstream) input topic, there is an associated (downstream) notification topic for batch status notifications.  
+
+[API specification on GitHub](https://github.com/Alvearie/hri-api-spec/tree/master/notifications/batchNotification.json)
 
 ### Batches
-Health Record Datasets often have organizational requirements to be processed together "as a set" (partially or in their entirety) when moving the data into a new cloud instance. Hence, HRI has been built with support to process a dataset as a _Batch_. See [Batch](glossary.md#batch) for a detailed definition.  
 
-How much data goes in a batch is really up to the Data Integrator. The Management API provides support for starting, completing, terminating, and searching for batches. Any change to a batch results in a message being written to the associated notification topic in Kafka. 
+The data often has organizational requirements to be processed together as a dataset called a [**batch**](glossary.md#batch). Processing can occur with either a partial batch or an entire batch. For these reasons, the Health Record Ingestion service has been built with to support batch dataset processing. 
```

Comment:
Does the 'batch' link need to be bold? None of the other glossary links are bold, so it looks strange.